### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by hamzacse**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by hamzacse — COMMENTED** [review]
Update the game over message so people know how to play again!

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (hanjoboiii) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment (hanjoboiii) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment (hanjoboiii) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

File: README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ _Collaborate and work together on GitHub._
   Do not use quotes on the <details> tag attributes.
 -->
 
-<details id=0 open>
+<details id=0>
 <summary><h2>Welcome</h2></summary>
 
 All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
```
Comment (hanjoboiii) [inline README.md:24]:
```suggestion
All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
```

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```
Comment (hanjoboiii) [inline index.html:47]:
```suggestions
  </tr>
```

File: README.md (hunk 2 above)
Comment (hamzacse) [inline README.md]:
```suggestion
All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.s
```

**Pull request merged**